src/helpers/UserStatus.js

```diff
@@ -1,20 +1,25 @@
 // @flow
 import * as React from 'react';
 
-import { withUser } from '../context/User';
+import { withUser, withSimpleToken } from '../context/User';
 import type { User } from '../types';
 
 type Props = {
   user: User,
+  simpleToken: ?string,
   children?: React.Node,
 };
 
-const RequireUserCriteria = (user_criteria: (user: User) => boolean) =>
-  withUser((props: Props) => {
-    if (!user_criteria(props.user)) return null;
-    return props.children;
-  });
+const RequireUserCriteria = () =>
```

Comment:
it didn't need to be function returning function if it's used just as `LoggedIn: RequireUserCriteria`

Reply:
...you can fix it in some following PR, no big deal